### DISCUSSION TITLE
Any automatic tilt correction tools for ct brain images

### DISCUSSION THREAD

**Discussion by avKostanov** [TorchIO-project/torchio#774] [Q&A]
Hello, is there any options to automatically correct tilt on brain ct images? Not nessesary in torchio (but it would be great!). Thank you in advance

**Comment by romainVala** [comment]
hello
what do you mean by 'correct the tilt'?
do you want to coregister the ct volume with an other one ?

**Reply by avKostanov** [reply]
I want brain on axial plane to be fully symmetric

**Comment by romainVala** [comment]
ok, I do not know an alogorithm to perform it directly with a given volume, but if you have a reference volume (with a similar contrast) which is fully symmetric, you then can use a realign between the 2 volumes to correct your tilt.
With 3d mri brain volume we usually use a template (smooth average of several brain, in a symmetric orientation) note that is is used not only to correct the orientation but to obtain all volume in a similar space.

Realign can  be then perform with traditional software ants, elastix, niftireg, freesurfer, fsl, mrtrix have  coregistration functions to perform this taks ...

(I am working on an elastix coregistration transform to perform it whithin torchio, but it is still work in progress ... )
